packages/xo-web/src/xo-app/host/tab-general.js

```diff
@@ -85,9 +85,9 @@ export default ({
         </Col>
         <Col mediumSize={3}>
           <p>
-            {host.license_params.sku_marketing_name} {host.version} ({
-              host.license_params.sku_type
-            })
+            {host.productBrand} {host.version} ({host.license_params.sku_type
+              ? host.license_params.sku_type
+              : 'GPLv2'})
```

Comment:
Why isn't `license_params.sku_type` correctly set to *GPLv2`?

Reply:
Because there is no license field filled in XCP-ng, for reasons beyond my knowledge right now.

Reply:
Then we should check against the `productBrand` instead.

Reply:
This is the license type (Free, Enterprise, whatever Citrix use) it's not related to the brand here (XenServer vs XCP-ng)

Reply:
It's related to the brand, because we should display `GPLv2` if `XCP-ng` otherwise we display `license_params.sku_type`.

Reply:
Fine for me (@pdonias go ahead)